src/transformers/models/resnet/configuration_resnet.py

```diff
@@ -68,7 +68,7 @@ class ResNetConfig(PretrainedConfig):
 
     >>> # Initializing a model (with random weights) from the resnet-50 style configuration
     >>> model = ResNetModel(configuration)
-    
+
```

Comment:
You are faster than my comment

Reply:
Noticed a few tests failing so ran it; should work nicely now 😄